xml/System.Net.Security/CipherSuitesPolicy.xml

```diff
@@ -65,9 +75,19 @@
         <ReturnType>System.Collections.Generic.IEnumerable&lt;System.Net.Security.TlsCipherSuite&gt;</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
-        <value>To be added.</value>
-        <remarks>To be added.</remarks>
+        <summary>Gets the collection of cipher suites passed through <see cref="M:System.Net.Security.CipherSuitesPolicy.#ctor(System.Collections.Generic.IEnumerable{System.Net.Security.TlsCipherSuite})" />, which are the ones allowed to negotiate with on a client or server <see cref="T:System.Net.Security.SslStream" /> authentication.</summary>
```

Comment:
`List of the cipher suites passed into the underlying implementation. It is a subset of passed in allowedCipherSuites`.

(for OSX this will equal to passed in list, for OpenSSL it might be smaller than passed in - implementation on OSX might be later improved so I wouldn't document that implementation detail)

Reply:
@krwq Style guidelines are that a summary is one sentence, and for a get-only property starts with "Gets ".  (get/set also start with Gets, but "Gets or sets" :smile:)

Reply:
Didn't know that 😄 my comment was mostly about the second part that the list is guaranteed to be a subset of a list but might not be exactly passed in list.

Reply:
And you can also use the Insert a suggestion button to provide the exact change that you want that they can commit directly:

![image](https://user-images.githubusercontent.com/12971179/61832525-43250280-ae26-11e9-9002-f464155f9d92.png)

Reply:
What about this? 
Also, in the Remarks section, we may want to explain what's the criteria used to filter `allowedCipherSuites`.
```suggestion
        <summary>Gets the subset of valid cipher suites passed into <see cref="M:System.Net.Security.CipherSuitesPolicy.#ctor(System.Collections.Generic.IEnumerable{System.Net.Security.TlsCipherSuite})" /> constructor.</summary>
```